crates/nu-cmd-extra/src/extra/math/cos.rs

```diff
@@ -54,7 +54,7 @@ impl Command for SubCommand {
         vec![
             Example {
                 description: "Apply the cosine to π",
-                example: "math pi | math cos",
+                example: "3.141592 | math cos | math round --precision 4",
```

Comment:
Grr, I see why it is necessary but I dislike the impact on the educational value of those examples, when they get stuffed with evermore unrelated parts to let them work as tests.

Reply:
agree :thinking:

Reply:
@sholderbach
is there a way to allow these example tests to `use std math PI`?

Reply:
also we'd need the use of `math round` because the precision of Nushell floats is not enough for commands like `math sin` to evaluate `PI` to integers :eyes: